examples/remote-proxy.php

```diff
@@ -6,13 +6,13 @@
 
 use ProxyManager\Factory\RemoteObject\Adapter\XmlRpc;
 use ProxyManager\Factory\RemoteObjectFactory;
-use Zend\Http\Client\Adapter\Exception\RuntimeException;
-use Zend\XmlRpc\Client;
+use Laminas\Http\Client\Adapter\Exception\RuntimeException;
+use Laminas\XmlRpc\Client;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-if (! class_exists('Zend\XmlRpc\Client')) {
-    echo "This example needs Zend\\XmlRpc\\Client to run. \n In order to install it, "
+if (! class_exists('Laminas\XmlRpc\Client')) {
+    echo "This example needs Laminas\\XmlRpc\\Client to run. \n In order to install it, "
         . "please run following:\n\n"
         . "\$ php composer.phar require zendframework/zend-xmlrpc:2.*\n\n";
```

Comment:
Should this line be changed as well?

Reply:
The Class can't exists if it's not imported and we can't import it without having it in composer so I believe it should.

However, the require command still mentions zendframework. This should be changed too

Reply:
Handled in #550